test/stat/stat_test.go

```diff
@@ -69,19 +70,67 @@ func TestCliStatForConduitNamespace(t *testing.T) {
 		t.Fatal(err.Error())
 	}
 
+	// test `conduit stat ns`
+	err = TestHelper.RetryFor(20*time.Second, func() error {
+		out, err := TestHelper.ConduitRun("stat", "ns")
+		if err != nil {
+			t.Fatalf("Unexpected stat error: %v", err)
+		}
+
+		rowStats, err := parseRows(out, 2, true)
+		if err != nil {
+			return err
+		}
+
+		conduitNsFound := false
+		emojivotoNsFound := false
+		for _, row := range rowStats {
+			if row.name == "emojivoto" {
```

Comment:
`emojivoto` isn't deployed to the test environment by default, so we can't assume that it's there. I think we could simplify this test a bit and make it more reliable by only checking for stats for the namespace where conduit is deployed, rather than contending with all namespaces. Something like:

```
t.Run("test conduit stat namespace", func(t *testing.T) {
	err := TestHelper.RetryFor(20*time.Second, func() error {
		out, err := TestHelper.ConduitRun("stat", "ns", TestHelper.GetConduitNamespace())
		if err != nil {
			t.Fatalf("Unexpected stat error: %v", err)
		}

		rowStats, err := parseRows(out, 1)
		if err != nil {
			return err
		}

		return validateRowStats(TestHelper.GetConduitNamespace(), rowStats, "4/4")
	})
	if err != nil {
		t.Fatal(err.Error())
	}
})
```

That would also allow you to undo most of the changes that you made in the `parseRows` func. And it allows you to use the `validateRowStats` func for validating namespace stats in addition to deploy stats. You would need to update that func to parameterize the expected mesh count, however, since it won't be the same for conduit deploys ("1/1") and the conduit namespace ("4/4"):

```
func validateRowStats(name string, rowStats map[string]*rowStat, expectedMeshCount string) error {
```

Reply:
ah whoops! I assumed it was. Yeah, it's probably a good idea to just check the installed conduit namespace!